unity-renderer/Assets/DCLPlugins/FallbackAssetLoaderPlugin/FallbackFontsLoaderPlugin.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using System.Collections.Generic;
+using TMPro;
+
+namespace DCLPlugins.FallbackFontsLoader
+{
+    public class FallbackFontsLoaderPlugin : IPlugin
+    {
+        private readonly string FALLBACK_FONTS_ADDRESS = "Assets/Fonts/Fonts Sources/";
+        private readonly List<string> FALLBACK_FONTS_TO_LOAD = new () { "NotoSansJP-SemiBold SDF.asset", "NotoSansSC-SemiBold SDF.asset", "NotoSansKR-SemiBold SDF.asset" };
```

Comment:
change to static as it should not be an instance field

Reply:
Shouldn't it be const as `FALLBACK_FONT_ADDRESS`?

Reply:
Done!